src/mainboard/raptor-cs/talos-2/Makefile.inc

```diff
@@ -3,3 +3,4 @@
 bootblock-y += bootblock.c
 romstage-y += cbmem.c
 romstage-y += romstage.c
+romstage-y += ../../../soc/ibm/power9/istep_14_2.c
```

Comment:
@krystian-hebel For some reason, I cound't make it work without this line. It would be great if You could take a look, what have I done wrong with makefile's.

Reply:
That's because `talos_2_support` doesn't have SoC Makefile's fixes merged yet. Can you rebase on top of `isteps_13_rebased`? That way you wouldn't have to implement RMW versions of SCOM over and over again.

Reply:
I have rebased the branch and makefiles are now working correctly. I have also got rid of mrw scom's.